fuel-relayer/src/mock_db.rs

```diff
@@ -1,11 +1,10 @@
+#![allow(missing_docs)]
```

Comment:
I haven't seen this lint tag needed before, is there a reason it hasn't shown before but is now an issue?

Reply:
It's because I have `#![deny(missing_docs)]` in the lib.rs but I don't want to force docs on test code.